### DISCUSSION TITLE
[RESOLVED] Open codeql database on startup

### DISCUSSION THREAD

**Discussion by CaledoniaProject** [github/codeql#8417] [Q&A]
When I open `vscode-codeql-starter.code-workspace` there's no database:

![1](https://user-images.githubusercontent.com/1357701/158042786-71d55aa7-9e05-49b6-8109-44f7b2b2eea3.jpg)

I want to modify the workspace and open a local database automatically. 
How can I do that?

**Comment by hvitved** [comment]
I don't think that is possible. However, if you add a database to your workspace, then it should still be there after you close and reopen the workspace.

**Reply by CaledoniaProject** [reply]
This one worked for me.

**Comment by CaledoniaProject** [comment]
How can I add a database to the workspace? I've dragged the database folder to "EXPLORER" and it's there on the next startup.
But codeql still won't open any database on startup.

**Reply by hvitved** [reply]
You click one of the buttons in your screenshot.
